test/locations.jl

```diff
@@ -10,5 +10,11 @@ using DataDeps: try_determine_load_path, determine_save_path, try_determine_pack
         @test try_determine_package_datadeps_dir(fn) == nothing
     end
 
-end
+    mktemp() do fn, fh
+        ENV["DATADEPS_LOAD_PATH"] = ""
+        ENV["DATADEPS_NO_STANDARD_LOAD_PATH"] = false
```

Comment:
`DATADEPS_NO_STANDARD_LOAD_PATH`
should be set to "true".
As this disables the standard locations

(sorry I said it wrong in the issue).

Also use `withenv(...) do` so that the enviroment variable setting is scoped.
Or it will break all the other tests, since they will not have anywhere to save.

Reply:
Check check